### PR TITLE
Add pendulum Bilinear example

### PR DESCRIPTION
**Pull request by bjack205**
Adds a bunch of stuff to work with EDMD models.

### CODE REVIEW

File: dev/bilinear_pendulum [hunk 1]
```diff
@@ -0,0 +1,79 @@
+
+import Pkg; Pkg.activate(@__DIR__)
```
Comment (bjack205) [inline dev/bilinear_pendulum:2]: Fairly complete example of using both models with ALTRO

File: dev/cartpole.jl [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import Pkg; Pkg.activate(@__DIR__)
```
Comment (bjack205) [inline dev/cartpole.jl:1]: Cartpole example that doesn't work right now

File: examples/learned_models/pendulum.jl [hunk 3]
```diff
@@ -0,0 +1,141 @@
+import Pkg; Pkg.activate(joinpath(@__DIR__, ".."))
```
Comment (bjack205) [inline examples/learned_models/pendulum.jl:1]: New script to generate the learned pendulum data

File: examples/models/cartpole_model.jl [hunk 4]
```diff
@@ -0,0 +1,51 @@
+struct BilinearCartpole <: RD.DiscreteDynamics
```
Comment (bjack205) [inline examples/models/cartpole_model.jl:1]: This has been pretty much been superseded by `EDMDModel`

**Review by jhlee0997 — APPROVED** [review]

**Pull request merged**